attributes/default.rb

```diff
@@ -24,28 +17,18 @@
 # Will be populated automaticaly when roles use, unless overrided.
 default['stackstorm']['components'] = %w(st2common)
 default['stackstorm']['service_binary'] = {}
+default['stackstorm']['install_stackstorm']['version'] = '1.4.0-8'
```

Comment:
That's cool, but approach is kinda not very cozy, when a build is updated you will need to update attribute somehow, that's harsh. I suggest to make installation pin-able to the last version or better to major version such as `1.4` in this case, this will keep st2 installs on user's machines coherent to major version.